internal/parser/parser.go

```diff
@@ -20,8 +20,54 @@ type Parser struct {
 	// subcontrol with enhancements
 	simpleSubCtrl *regexp.Regexp
 	// subcontrol with enhancements
-	subCtrlEnh *regexp.Regexp
-	data       map[controlFamily]map[string]v3c.Satisfies
+	subCtrlEnh     *regexp.Regexp
+	// subcontrol with extra enhancements
+	subCtrlEnhPlus *regexp.Regexp
+	data           map[controlFamily]map[string]v3c.Satisfies
+}
+
+// parseFamily takes a string and returns an OpenControl friendly format for NIST Family.
+func parseFamily(family string) string {
```

Comment:
These families are not reflected anywhere in compliance masonry, right? Would be good to get these from somewhere as a constant.

Reply:
This came from the OpenControl standards repo
https://github.com/opencontrol/standards/blob/master/utils/parser.go#L142

Reply:
Since this was not an exported package, I simply modified it. We can extract this out into its own utils in our repo as well.. What do you think?